input.go

```diff
@@ -737,6 +746,7 @@ type CustomActionsTriggerDefinitionUpdateInput struct {
 
 // CustomActionsWebhookActionCreateInput Specifies the input fields used in the `customActionsWebhookActionCreate` mutation
 type CustomActionsWebhookActionCreateInput struct {
+	Async          bool                        `json:"async" yaml:"async" example:"false"`                                                                                                                             // Whether the action expects an additional, asynchronous response upon completion (Required)
```

Comment:
This isn't breaking because it has the default of false when not present

Reply:
I think that means it needs a be a pointer in golang so we have 3 states
- not present
- false
- true